templates/repo/create.tmpl

```diff
@@ -24,10 +24,10 @@
 									{{.SignedUser.ShortName 20}}
 								</div>
 								{{range .Orgs}}
-									<div class="item" data-value="{{.ID}}" title="{{.Name}}">
-										<img class="ui mini image" src="{{.RelAvatarLink}}">
-										{{.ShortName 20}}
-									</div>
+								<div class="item" data-value="{{.ID}}" title="{{.Name}}">
```

Comment:
As far as I can see, other templates follow style of additional indentation, is there any specific reason why this PR changes this on this file only?